fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -257,6 +257,171 @@ sebasedBiasFieldMNI="$SubjectFolder/$AtlasSpaceFolder/Results/$NameOffMRI/${Name
+                                                                         # e.g. --slicetimerparams="--odd@--ocustom=<CustomInterleaveFile>"
+BOLDMask=`opts_GetOpt1 "--boldmask" $@`                                  # Specifies what mask to use for the final bold:
+                                                                         #   T1_fMRI_FOV: combined T1w brain mask and fMRI FOV masks (the default), 
+                                                                         #   T1_DILATED_fMRI_FOV: a once dilated T1w brain based mask combined with fMRI FOV
+                                                                         #   T1_DILATED2x_fMRI_FOV: a twice dilated T1w brain based mask combined with fMRI FOV, 
+                                                                         #   fMRI_FOV: a fMRI FOV mask
+                                                                         #   NOTE: When using SEBASED bias field correction, the mask will default to T1_fMRI_FOV
+                                                                         #   as the bias field is masked already.
+fMRIReference=`opts_GetOpt1 "--fmriref" $@`                              # Reference BOLD run name (i.e., --fmriname from run to be used as *reference*) to use as 
+                                                                         #   motion correction target and to copy atlas (MNI152) registration from (or NONE; default).
+                                                                         #   NOTE: The reference BOLD has to have been fully processed using fMRIVolume pipeline, so
+                                                                         #   that a distortion correction and atlas (MNI152) registration solution for the reference
+                                                                         #   BOLD already exists. Also, the reference BOLD must have been acquired using the same
+                                                                         #   phase encoding direction, or it can not serve as a valid reference. 
+fMRIReferenceReg=`opts_GetOpt1 "--fmrirefreg" $@`                        # In the cases when BOLD image is registered to a specified BOLD reference, this option 
+                                                                         #   specifies whether to use 'linear' or 'nonlinear' registration to reference BOLD.
+                                                                         #   Default is 'linear'.
+
+
+
+# Defaults
+PreregisterTool=`opts_DefaultOpt $PreregisterTool "epi_reg"`
+DoSliceTimeCorrection=`opts_DefaultOpt $DoSliceTimeCorrection "FALSE"`   # WARNING: This LegacyStyleData option of slice timing correction is performed before motion correction 
+                                                                         #   (as is typically done in legacy-style brain imaging) and thus assumes that the brain is motionless. 
```

Comment:
I don't know if I would imply that there is consensus on the order issue, given that AFNI and SPM apply STC prior to MC, and FSL does it opposite.  See brief discussion here https://wiki.humanconnectome.org/display/PublicData/HCP+fMRI+slice-timing+acquisition+parameters and the links therein.

Reply:
@mharms this is text as provided by @glasserm. I'm ok to include what is your consensus.

Reply:
Well there are problems with both approaches (which is why there is no consensus), so the point of the warning is that slice timing correction as currently implemented is problematic.

Reply:
My point is that it isn't accurate to say "(as is typically done in legacy-style brain imaging)", and we should just drop that parenthetical.  The entire rest of the warning is still fine as is.

Reply:
Ok. I removed te parenthetical.